README.md

```diff
@@ -62,10 +62,9 @@ The video will sync automatically when the host plays/pauses the video or seeks
 
 ### To start the server locally : 
 
- 1. Install node on your local from this link :  https://nodejs.org/en/download/
- 2. Open command line and install npm using this command : **npm install -g npm**
- 3. Open the "server" folder that we downloaded earlier.
- 4. Open command line there and run this command :    **node index.js**  
+ 1. Open the "server" folder and open command line there.
+ 2. Run this command to install dependencies : **npm i cors ejs express socket.io**
```

Comment:
You just need to write "npm install", package.json file has all the dependencies which are necessary to be installed